covidX/settings/BUILD

```diff
@@ -30,7 +30,6 @@ py_library(
         "//apps/auth_zero:apps",
         "//apps/auth_zero:models",
         "//apps/auth_zero:auth0backend",
-        "//:__init__",
```

Comment:
This should be changed to reference the `__init__.py` file in the `covidX/settings` directory instead of removing it altogether. See [the previous comment](https://github.com/Xcov19/covidX/pull/124/files#r629565802).